core/src/main/resources/META-INF/primefaces-extensions.taglib.xml

```diff
@@ -10414,5 +10414,68 @@ See showcase for an example. Default null.]]>
             <type>java.lang.String</type>
         </attribute>
     </tag>
+    <tag>
+        <description><![CDATA[ClockPicker based on https://weareoutman.github.io/clockpicker/]]></description>
+        <tag-name>clockpicker</tag-name>
+        <component>
+            <component-type>org.primefaces.extensions.component.ClockPicker</component-type>
+            <renderer-type>org.primefaces.extensions.component.ClockPickerRenderer</renderer-type>
+        </component>
+        <attribute>
+            <description><![CDATA[Unique identifier of the component in a namingContainer.]]></description>
+            <name>id</name>
+            <required>false</required>
+            <type>java.lang.String</type>
+        </attribute>
+        <attribute>
+            <description><![CDATA[Boolean value to specify the rendering of the component, when set to false component will not be rendered.]]></description>
+            <name>rendered</name>
+            <required>false</required>
+            <type>java.lang.Boolean</type>
+        </attribute>
+        <attribute>
+            <description><![CDATA[Value of the component.]]>
+            </description>
+            <name>value</name>
+            <required>false</required>
+            <type>java.time.LocalTime</type>
```

Comment:
Shouldn't we keep this an Object? You are able to use a converter.

Reply:
Yes, you are right. I reverted the change from LocalTime back to Object.